ivy/functional/backends/tensorflow/experimental/statistical.py

```diff
@@ -6,6 +6,25 @@
 from ivy.func_wrapper import with_supported_dtypes
 from . import backend_version
 
+def histogramdd(
+        input: Union[tf.Tensor, tf.Variable],
+        /,
+        *,
+        bins: Optional[Union[Tuple[int],int,Tuple[float]]] = 10,
+        weights : Optional[Tuple[int]] = None,
+        range: Optional[Union[Tuple[int], int]] = [0., 1., 0., 1.],
+        density: Optional[bool] = False
+) -> Union[tf.Tensor, tf.Variable]:
```

Comment:
Kindly keep the return type consistent in all files. Remove `tf.Variable` and add `Tuple` in the beginning instead like you have done in other files.